postprocessor.nim

```diff
@@ -20,10 +20,25 @@ proc pp(n: var PNode) =
       m.add n
       n = m
   of nkInfix, nkPrefix, nkPostfix:
-    for i in 1.. < n.safeLen: pp(n.sons[i])
+    for i in 1.. < n.safeLen: pp(n.sons[i], stmtList)
   of nkAccQuoted: discard
+
+  of nkStmtList:
+    for i in 0.. < n.safeLen: pp(n.sons[i], n)
+  of nkRecList:
+    var consts: seq[int] = @[]
+    for i in 0.. < n.safeLen:
+      if n.sons[i].kind == nkConstSection:
+        consts.insert(i)
```

Comment:
Nice catch. I've fixed this, and also added a second `idx` argument to keep track of where in the stmtList the constSection should be inserted.

Is it safe to modify the stmtList.sons sequence during iteration?